resources/compass/charts/gateway/templates/virtualservice.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: {{ template "fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  hosts:
+    - '{{ .Values.virtualService.host }}.{{ .Values.global.ingress.domainName }}'
+  gateways:
+    - {{ .Values.global.istio.gateway.name }}
```

Comment:
You should use existing virtual service from kyma-system here. As far as I know you cannot register two gateways in istio for the same domain (in this case for example *.kyma.local).